src/debugging.ts

```diff
@@ -5,7 +5,7 @@ import { identityToAddress, SendJob } from "./multichainhelpers";
 
 /** A string representation of a coin in a human-readable format that can change at any time */
 export function debugCoin(coin: BcpCoin): string {
-  return `${coin.whole + coin.fractional / 10 ** coin.sigFigs} ${coin.tokenTicker}`;
+  return `${coin.quantity} ${coin.tokenTicker}`;
```

Comment:
This should be converted to whole coins, as quantity is measired in atoms, not in whole coins.